_config.yml

```diff
@@ -101,18 +101,29 @@ defaults:
 include:
   # dotfiles are excluded by default
   - .well-known
+  - docs/accessibility-assessment-ial1-account-creation.pdf
+  - docs/accessibility-assessment-ial2-identity-verification-process.pdf
 
 exclude:
+  - assets/img/favicons
+  - assets/js/build/main.js.LICENSE.txt
```

Comment:
Wondering if we should keep this if there was any licensing obligations. That being said, I think the way it's generated is from JS comments in the original source files, so probably neither consistent nor the canonical licensing information.

If we wanted to keep, maybe `.LICENSE.txt` could be an allowed "extension" ?

Reply:
I vote to keep excluding it for now, I don't think it's meaningful content. And if we did have important license content, I think we should make it navigable at least